coreneuron/network/tqueue.hpp

```diff
@@ -65,9 +65,10 @@ extern void splay(SPBLK*, SPTREE*);    /* reorganize tree */
 extern SPBLK* sphead(SPTREE*);         /* return first node in tree */
 extern void spdelete(SPBLK*, SPTREE*); /* delete node from tree */
 
+struct DiscreteEvent;
 class TQItem {
   public:
-    void* data_ = nullptr;
+    DiscreteEvent* data_ = nullptr;
```

Comment:
Just to be sure: all event types are derived from `DiscreteEvent` class and hence the change, right?

Reply:
Yes..this was a change I made while debugging, and it seems like a straightforward improvement so I left it in.